.vscode/settings.json

```diff
@@ -16,7 +16,4 @@
     "files.watcherExclude": {
         "**/target": true
     },
-    "rust-analyzer.cargo.features": [
```

Comment:
Latest Rust Analyzer update complains about `python` not being a recognized feature on some crates.